testing/trino-tests/src/test/java/io/trino/execution/TestGrantOnTable.java

```diff
@@ -105,8 +105,8 @@ public void testValidGrantWithGrantOption(String privilege)
         queryRunner.execute(admin, format("GRANT %s ON TABLE table_one TO %s WITH GRANT OPTION", privilege, username));
 
         assertThat(assertions.query(user, "SHOW TABLES FROM default")).matches("VALUES (VARCHAR 'table_one')");
-        assertThat(assertions.query(user, format("GRANT %s ON TABLE table_one TO %s", privilege, randomUsername()))).matches("VALUES (BOOLEAN 'TRUE')");
-        assertThat(assertions.query(user, format("GRANT %s ON TABLE table_one TO %s WITH GRANT OPTION", privilege, randomUsername()))).matches("VALUES (BOOLEAN 'TRUE')");
+        assertions.query(user, format("GRANT %s ON TABLE table_one TO %s", privilege, randomUsername()));
```

Comment:
`assertions.query` returns `AssertProvider` and must not be used without `assertThat`
The fact that it runs query during construction should be viewed as an impl detail

Use `[QueryAssertions.]assertQuerySucceeds(queryRunner, session, sql)` here

Reply:
`assertQuerySucceeds` has protected access modified and thus this test class can't access it. I will use `assertUpdate`